src/share/poudriere/include/pkg.sh

```diff
@@ -417,6 +421,8 @@ delete_pkg_xargs() {
 	# Delete the package and the depsfile since this package is being deleted,
 	# which will force it to be recreated
 	{
+		# If ${pkg} is a symlink, delete the target as well
+		[ -L "${pkg}" ] && echo $(realpath "${pkg}")
```

Comment:
Please avoid `&&` as if it ends up being the last in a statement the statement can return non-zero with `set -e`. So we avoid it as a pattern because it sneaks in eventually.
```
# sh -c 'set -o pipefail; set -e; dolink() { [ -L / ] && true; }; dolink; echo done'; echo $?
1